cogs/utils/dataIO.py

```diff
@@ -9,6 +9,15 @@ class InvalidFileIO(Exception):
 class DataIO():
     def __init__(self):
         self.logger = logging.getLogger("red")
+        try:
+            __import__("aiofiles")
+            import aiofiles
```

Comment:
can you explain what the `__import__()` is used for?
I'm curious as to why you used both